homeassistant/components/airnow/sensor.py

```diff
@@ -86,7 +86,11 @@ def __init__(
     @property
     def native_value(self):
         """Return the state."""
-        self._state = self.coordinator.data[self.entity_description.key]
+        if self.entity_description.key in self.coordinator.data:
+            self._state = self.coordinator.data[self.entity_description.key]
+        else:
+            self._state = None
```

Comment:
Could this be simplified?

```suggestion
        self._state = self.coordinator.data.get(self.entity_description.key)
```

Reply:
Yeah, that seems better.